<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpressionSelect</class>
 <widget class="QGroupBox" name="ExpressionSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelSearch">
     <property name="text">
      <string>Search for:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="CustomLineEdit" name="lineEditSearch"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelReplace">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditReplace"/>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="radioButtonRegex">
     <property name="text">
      <string>Regex</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="radioButtonPlain">
     <property name="text">
      <string>Plain</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="checkBoxCaseSensitive">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
